projects/plugins/jetpack/tools/webpack.config.search.js

```diff
@@ -7,11 +7,28 @@ const {
 	defaultRequestToExternal,
 	defaultRequestToHandle,
 } = require( '@wordpress/dependency-extraction-webpack-plugin/util' );
+const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const webpack = require( 'webpack' );
 
 const isDevelopment = process.env.NODE_ENV !== 'production';
 
+/**
+ * As the version number in package.json deesn't have the same format as it is in PHP.
+ * So we extract the Jetpack version from plugin PHP file and to make sure Webpack
+ * gets exactly the same.
```

Comment:
```suggestion
 * Since the version number in package.json doesn't match PHP's `JETPACK__VERSION` constant,
 * we rely on extracting the `JETPACK__VERSION` constant directly from the plugin PHP file
 * for use in our Webpack configuration. 
```